share_url.go

```diff
@@ -0,0 +1,42 @@
+package yoti
+
+import (
+	"encoding/json"
+)
+
+var (
+	// ShareURLHTTPErrorMessages specifies the HTTP error status codes used
+	// by the Share URL API
+	ShareURLHTTPErrorMessages = map[int]string{
+		400: "JSON is incorrect, contains invalid data: %[2]s",
+		404: "Application was not found: %[2]s",
+	}
+)
+
+// ShareURL contains a dynamic share qr code
```

Comment:
Same here with consistent capitalisation. My preference would be `QR code`
```suggestion
// ShareURL contains a dynamic share QR code
```